### DISCUSSION TITLE
Unlit Material + Normals?

### DISCUSSION THREAD

**Discussion by ss9326** [google/filament#8279] [Q&A]
I would like a material that doesn't interact with lighting, but still gives access to worldNormals so I can check its facing direction in the shader.

So either an unlit material with worldNormals, OR a lit material with lighting disabled.

How can I do this?

**Comment by SC5Shout** [comment]
I believe that using this function `MaterialBuilder& require(VertexAttribute attribute) noexcept;` with `VertexAttribute::TANGENTS` attribute set will make normals available
